web/src/pages/mgmt/modifyBusiness.tsx

```diff
@@ -0,0 +1,208 @@
+                    onChange={(event) => setLicenseStatus(event.target.value)}
+                  >
+                    {licenseStatuses.map((licenseStatus) => (
+                      <MenuItem key={licenseStatus} value={licenseStatus}>
+                        {licenseStatus}
+                      </MenuItem>
+                    ))}
+                  </Select>
+                </FormControl>
+              </div>
+            </div>
+            <div className="flex margin-top-4">
+              <PrimaryButton
+                isColor="primary"
+                onClick={() => {
+                  if (!!licenseStatus && !!licenseName && !!business) {
+                    updateQueue?.queueBusiness({
+                      ...business,
+                      taskProgress: {
+                        ...business.taskProgress,
+                        [taskID]: "COMPLETED",
+                      },
+                      licenseData: {
+                        licenses: {
+                          ...business?.licenseData?.licenses,
+                          [licenseName]: generateLicenseDetails({
+                            expirationDateISO: getCurrentDate().add(1, "month").toISOString(),
+                            licenseStatus: licenseStatus as LicenseStatus,
+                          }),
+                        },
+                        lastUpdatedISO: getCurrentDate().toISOString(),
+                      },
+                    });
+
+                    setLicenseStatus("");
+                    setLicenseName("");
+                    setTaskId("");
+                    updateQueue?.update();
+                  }
+                }}
+              >
+                Add License
+              </PrimaryButton>
+              <SecondaryButton
+                isColor="primary"
+                onClick={() => {
+                  if (business?.licenseData) {
+                    const taskProgress = licenseSearchEnabledTaskIds.reduce(
+                      (acc: Record<string, TaskProgress>, curr: string): Record<string, TaskProgress> => {
+                        acc[curr] = "NOT_STARTED";
+                        return acc;
+                      },
+                      {}
+                    );
+
+                    updateQueue?.queueBusiness({
+                      ...business,
+                      taskProgress: {
+                        ...business.taskProgress,
+                        ...taskProgress,
+                      },
+                      licenseData: undefined,
+                    });
+
+                    setLicenseStatus("");
+                    setLicenseName("");
+                    setTaskId("");
+                    updateQueue?.update();
+                  }
+                }}
+              >
+                Delete All License Data
+              </SecondaryButton>
+            </div>
+          </div>
+        </SingleColumnContainer>
+      </main>
+    </PageSkeleton>
```

Comment:
I'm suprised that when I go to the license and look at the check status tab I don't see my expired state?

<img width="1145" alt="image" src="https://github.com/user-attachments/assets/6a79592e-403e-452c-9626-82e9c5bc85ae">
<img width="1635" alt="image" src="https://github.com/user-attachments/assets/879b947d-8bd4-4200-9498-7621438d2b7a">

did I not set this up correctly? (I did set the license status before logging in)

Reply:
I'll relay this to Dillon. It's not a bug, the tickets were created to enable it in certain places.